relay/src/main.rs

```diff
@@ -176,10 +191,12 @@ async fn main() -> anyhow::Result<()> {
             eth_node,
             eth_chain_id,
             private_key,
+            connection_retry_attempts,
+            connection_retry_interval,
         } => {
             let relayer = Relayer {
                 rest_api: true,
-                dev_mode,
+                dev_mode: dev_mode,
```

Comment:
I think clippy would suggest keeping this just `dev_mode,` any reason to not use the alias assignment?